frontend/app/src/components/helper/CopyTooltip.vue

```diff
@@ -1,15 +1,32 @@
 <script setup lang="ts">
-defineProps<{
+import { PriceOracle } from '@/types/settings/price-oracle';
+
+const props = defineProps<{
   copied: boolean;
   tooltip?: string | null;
   disabled?: boolean;
+  oracle?: string;
 }>();
 
 defineSlots<{
   default: () => any;
 }>();
 
 const { t } = useI18n();
+
+const oracleText = computed<string | undefined>(() => {
+  const oracleVal = props.oracle;
+  if (!oracleVal)
+    return undefined;
+
+  const mapping: Record<string, string> = {
+    [PriceOracle.MANUALCURRENT]: t('oracles.manual_current'),
+    [PriceOracle.UNISWAP2]: t('oracles.uniswap_v2'),
+    [PriceOracle.UNISWAP3]: t('oracles.uniswap_v3'),
+  };
+
+  return mapping[oracleVal] || oracleVal;
+});
```

Comment:
Can we do it in a way where the `CopyTooltip` component is not aware of the oracles?

Maybe we introduce a slot to the component so that if we need custom content we can add it in the slot.